x-pack/plugins/uptime/common/translations.ts

```diff
@@ -37,3 +37,60 @@ export const MonitorStatusTranslations = {
     defaultMessage: 'Alert when a monitor is down or an availability threshold is breached.',
   }),
 };
+
+export const TlsTranslations = {
```

Comment:
Moved alert translations to common, in order to use them within server to derive reason message.